Makefile

```diff
@@ -48,9 +48,8 @@ lint:
 		flake8 $(ALL_PY_SRCS) && \
 		mypy $(PY_MODULE) && \
 		bandit -c pyproject.toml -r $(PY_MODULE)
-# TODO: Reenable interrogate
-#		interrogate -v -c pyproject.toml . && \
-#	        git diff --exit-code
+
+	  git diff --exit-code
```

Comment:
I'm not a fan of this because it makes linting fail when attempting to lint unstaged changes. I think we should be running `black --check` instead.

Reply:
Ah, yep. Okay, I'll do a follow-on to drop this and use `black --check` instead.

Reply:
(And maybe add a `make reformat` command that actually runs the autoformatters we use)